.github/workflows/node.js.yml

```diff
@@ -25,5 +25,6 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build --if-present
+    - run: npm install -g gatsby-cli
```

Comment:
I think using yarn will be better @endurance21

Reply:
does not really matter much , as those installed packages are gone once tasks are done on the Vm of github servers .

Reply:
it does matter, as `npm install` will look for `package-lock.json` (https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json) while `yarn install` uses `yarn.lock`. As we have the latter checked in, we should use `yarn install` in the CI.

Reply:
And why do we install `gatsby-cli` globally? What is that used for, and why is not a dependency of this project?

Reply:
Actually we assume Gatsby-cli to be a independent tool.
And we can easily run it from any path.

Also if we use `gatsby-cli`  as dependency , we would a task runner like `npx` to run Gatsby like 
` npx Gatsby build ` or smth.

Reply:
> it does matter, as `npm install` will look for `package-lock.json` (https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json) while `yarn install` uses `yarn.lock`. As we have the latter checked in, we should use `yarn install` in the CI.
You are right 
In the context of working they both are differrent.

What I meant is we can use either npm or yarn , it should not matter much on testing build process thing.